src/www/ui/api/documentation/openapi.yaml

```diff
@@ -654,6 +654,23 @@ paths:
         schema:
           type: boolean
           default: false
+      - name: page
+          description: Page number (starts from 1)
+          required: false
+          in: header
+          schema:
+            type: integer
+            default: 1
+            minimum: 1
+      - name: limit
+          description: Limits of responses per request
+          required: false
+          in: header
+          schema:
+            type: integer
+            default: 50
+            minimum: 1
+            maximum: 100
```

Comment:
For this particular endpoint, I think the max limit can be increased to 1000

Reply:
Corrected it